nixos/lib/make-ext4-fs.nix

```diff
@@ -9,13 +9,13 @@
 }:
 
 let
-  sdClosureInfo = pkgs.closureInfo { rootPaths = storePaths; };
+  sdClosureInfo = pkgs.buildPackages.closureInfo { rootPaths = storePaths; };
```

Comment:
This one really needs a proper solution rather sprinkling this `buildPackages.` in random places and being inconsistent with rest of Nixpkgs.

Reply:
@dezgeg Hm what do you suggest? It's only used via string interpolation; it's not depended on in a way that can be put into `nativeBuildInputs` AFAIK. Also, it's not harmful to be more explicit about `buildPackages`, is it?

Reply:
Ah oops, I was supposed to put the comment on the `nativeBuildInputs = with pkgs.buildPackages; ...` line. 

But in general, this `buildPackages.` thing is bad since it breaks overriding inputs (like in `callPackage ./foo { bar = baz; };`) so it would need a replacement one day.

Reply:
@dezgeg Ah, so I should take these inputs in the file's argument list and list them under `nativeBuildInputs` rather than maintaining the `with pkgs;` that was used before, right? TBH, I'm fairly confident I didn't need to add `.buildPackages` here....